src/promptflow-core/promptflow/_utils/flow_utils.py

```diff
@@ -70,9 +73,8 @@ def resolve_flow_path(
     :param base_path: The base path to resolve the flow path. If not specified, the flow path will be
       resolved based on the current working directory.
     :type base_path: Union[str, Path, PathLike]
-    :param new: If True, the function will return the flow directory path and the file name of the
-        target yaml that should be created. If False, the function will try to find the existing
-        target yaml and raise FileNotFoundError if not found.
+    :param check_flow_exist: If True, the function will return the flow directory path and the file name of the
```

Comment:
If True, the function will try to check the target yaml and raise FileNotFoundError if not found?

Reply:
done